README.md

```
@@ -218,4 +218,14 @@ Try it in your browser: [https://red.ht/its-hub-demo](https://red.ht/its-hub-dem
 
 To run the demo yourself, see the [demo setup instructions](https://github.com/lukeinglis/its_hub_demo/blob/main/demo_ui/README.md).
 
+## Demo
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)